docs/user/rstudio/ide/guide/publish/publishing.qmd

```diff
@@ -2,91 +2,60 @@
 title: "Publishing"
 ---
 
-adapted from: <https://docs.rstudio.com/connect/user/publishing/>
+::: {.callout-warning}
+This section is not complete until we have updated screenshots for publishing to Posit Connect.
+:::
 
-### How to install
+The following documentation is an overview of "Push-button Deployment" in RStudio. 
 
-Adapted from \<rsconnecthttps://shiny.rstudio.com/articles/shinyapps.html\>
+For full documentation on options for specific platforms:  
```

Comment:
Again, see comment/question about rstudio.cloud publishing

Reply:
Holding on these two sections for now, until we have clarity. Thanks!